src/sentry/dynamic_sampling/rules/biases/custom_rule_bias.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, cast
+
+from sentry.dynamic_sampling.rules.biases.base import Bias
+from sentry.dynamic_sampling.rules.utils import Condition, PolymorphicRule
+from sentry.models import CUSTOM_RULE_DATE_FORMAT, CustomDynamicSamplingRule, Project
+from sentry.utils import json
+
+
+class CustomRuleBias(Bias):
+    """
+    Boosts at 100% sample rate all the traces that have a replay_id.
+    """
+
+    def generate_rules(self, project: Project, base_sample_rate: float) -> List[PolymorphicRule]:
+
+        rules = CustomDynamicSamplingRule.get_project_rules(project)
+
+        ret_val: List[PolymorphicRule] = []
+
+        for rule in rules:
+            condition = json.loads(rule.condition)
+            ret_val.append(
+                {
+                    "samplingValue": {"type": "sampleRate", "value": rule.sample_rate},
+                    "type": "transaction",
```

Comment:
Are we sure we want to implement a `transaction` rule? From the user perspective this might lead to broken traces.

If we were to use a `trace` rule, we might need a more refined UX on the frontend though, since we will have to teach the user the idea of head of trace.

Reply:
Yes, it was decided we want transaction rules